### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by shaneutt**
I somehow left this out of my release PR, need to add this changelog for the v1.3.1 release.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -94,6 +94,17 @@ released and the release notes may change significantly before then.
 [go]:https://golang.org
 [ktf]:https://github.com/kong/kubernetes-testing-framework
 
+## [1.3.1] - 2021/06/03
```
Comment (hbagdi) [inline CHANGELOG.md:97]: Please define the link at the bottom of this file.

**Review by hbagdi — APPROVED** [review]
rubber stamp

File: CHANGELOG.md (hunk 1 above)
Comment (shaneutt) [reply CHANGELOG.md:97]: :+1:

**Review by hbagdi — APPROVED** [review]
:rubber stamp:

**Pull request merged**